ahead: 1
max_vel: 0.2
min_vel: 0.05
max_w_vel: 1.2
accel_range: 0.1
decel_range: 0.1
goal_range: 0.01
goal_yaw_range: 0.1